lua/entities/gmod_wire_freezer.lua

```diff
@@ -88,7 +88,7 @@ function ENT:UpdateOutputs()
 end
 
 function ENT:CheckEnt( ent )
-	if IsValid(ent) then
```

Comment:
I think this one needs to stay IsValid(ent) because there's a possibility of the function receiving nil